packages/components/src/font-size-picker/index.js

```diff
@@ -88,12 +89,11 @@ function FontSizePicker( {
 					/>
 				}
 				{ ( ! withSlider && ! disableCustomFontSizes ) &&
-					<input
-						className="components-range-control__number"
-						type="number"
-						onChange={ onChangeValue }
+					<NumberControl
```

Comment:
With this change the number input field is rendered inside a base control div, that creates an unexpected small design change:
<img width="561" alt="screenshot 2019-02-13 at 18 27 03" src="https://user-images.githubusercontent.com/11271197/52734493-4ab1c700-2fbd-11e9-9ef5-27e6579f3b1b.png">

The input field now is not in the middle of the reset and the size picker because of some padding.

Reply:
Must've missed that while consolidating the different components CSS. That's actually a margin on the input itself. Just fixed.